prospector/filter_rank/rank.py

```diff
@@ -124,3 +124,63 @@ def make_dataframe(
         )  # This line needs to be updated to correspond to
         # the actual features
     return
+
+
+def apply_rules(
```

Comment:
I would put this in a separate `rules.py` module, so that we keep it distinct from ML-based commit annotation